examples/tooltip.py

```diff
@@ -0,0 +1,131 @@
+
+from dash import Dash, dcc, html, Input, Output, no_update
+import pandas as pd
+import plotly.express as px
+import plotly.io as pio
+import dash_mantine_components as dmc
+
+
+pio.renderers.default = 'browser'
+
+# Reading data
+df = pd.read_table(
+    "https://raw.githubusercontent.com/plotly/datasets/master/global_super_store_orders.tsv"
+)
+
+# Transforming column types so that datetime functions can be applied correctly
+df["Order Date"] = pd.to_datetime(df["Order Date"])
+df["Profit"] = df["Profit"].str.replace(",", ".")
+df["Profit"] = df["Profit"].astype("float")
+
+
+# Transforming 'Order Date' column into yyyy-mm format
+df['year_month'] = pd.DatetimeIndex(df['Order Date']).to_period('M').astype(str)
+
+# Grouping data as per 'Order Date' dimension.
+# Getting time series - profit per month.
+df_grouped = df[['Order Date', 'Profit']].groupby(
+    by=pd.Grouper(
+        key='Order Date',
+        axis=0, freq='M'
+    )).sum().reset_index()
+
+# Creating base bar chart.
+# Why 'base'? --> Because it will be shown on page all the time.
+# Below there is one bar chart that will be shown only on hover.
+fig = px.bar(
+    data_frame=df_grouped,
+    x='Order Date',
+    y='Profit',
+    template="simple_white"
+)
+
+fig.update_layout(margin=dict(t=50, l=25, r=25, b=25))
+
+# turn off native plotly.js hover effects - make sure to use
+# hoverinfo="none" rather than "skip" which also halts events.
+fig.update_traces(hoverinfo="none", hovertemplate=None)
+
+fig.update_layout(
+    xaxis=dict(title='Year'),
+    yaxis=dict(title='Population'),
```

Comment:
You mean "Profit" ?
Actually the default X and Y titles of "Order Date" and "Profit" are pretty good.